org.eclipse.ui.views.pdf/plugin.xml

```diff
@@ -49,4 +49,110 @@
       </enablement>
     </renameParticipant>
   </extension>
-</plugin>
\ No newline at end of file
+  <extension
+        point="org.eclipse.ui.commands">
+     <command
+           categoryId="org.elysium.ui.commandCategories.LilyPond"
+           id="org.eclipse.ui.views.pdf.ZoomIn"
+           name="Zoom In">
+     </command>
+     <command
+           categoryId="org.elysium.ui.commandCategories.LilyPond"
+           id="org.eclipse.ui.views.pdf.ZoomOut"
+           name="Zoom Out">
+     </command>
+     <command
+           categoryId="org.elysium.ui.commandCategories.LilyPond"
+           id="org.eclipse.ui.views.pdf.ZoomNormal"
+           name="Zoom Normal">
+     </command>
+     <command
+           categoryId="org.elysium.ui.commandCategories.LilyPond"
+           id="org.eclipse.ui.views.pdf.ZoomPage"
+           name="Fit Page">
+     </command>
+     <command
+           categoryId="org.elysium.ui.commandCategories.LilyPond"
+           id="org.eclipse.ui.views.pdf.ZoomHeight"
+           name="Fit Height">
+     </command>
+     <command
+           categoryId="org.elysium.ui.commandCategories.LilyPond"
+           id="org.eclipse.ui.views.pdf.ZoomWidth"
+           name="Fit Width">
+     </command>
+     <command
+           categoryId="org.elysium.ui.commandCategories.LilyPond"
+           id="org.eclipse.ui.views.pdf.NavigateFirst"
+           name="First Page">
+     </command>
+     <command
+           categoryId="org.elysium.ui.commandCategories.LilyPond"
+           id="org.eclipse.ui.views.pdf.NavigateLast"
+           name="Last Page">
+     </command>
+     <command
+           categoryId="org.elysium.ui.commandCategories.LilyPond"
+           id="org.eclipse.ui.views.pdf.NavigatePrevious"
+           name="Previous Page">
+     </command>
+     <command
+           categoryId="org.elysium.ui.commandCategories.LilyPond"
+           id="org.eclipse.ui.views.pdf.NavigateNext"
+           name="Next Page">
+     </command>
+  </extension>
+  <extension
+        point="org.eclipse.ui.bindings">
+     <key
+           commandId="org.eclipse.ui.views.pdf.ZoomIn"
+           schemeId="org.eclipse.ui.defaultAcceleratorConfiguration"
+           sequence="CTRL++">
```

Comment:
Please substitute CTRL with M1 for platform independence (it corresponds to CMD on OS X).